.gitignore

```diff
@@ -20,6 +20,8 @@ Makefile.in
 
 .diags.log.meta
 
+.vscode/
```

Comment:
Removed.
I added this directly to the .gitignore to prevent myself from checking workspace setting files generated by VSCode. I'll talk yo Alan and Jason about where to put it, and do a separate pull request for it.